Индекс максимального числа в массиве

danqa03 opened the pull request:
Найти максимальное число в массиве, его индекс - и вывести все на консоль

danqa03 committed:
Индекс максимального числа в массиве

InTeam7 commented on Homework/Homework/Program.cs:
```diff
@@ -180,4 +183,20 @@ public static int GetElementCount(int[] arr, int itemToCount)
+    {
+        double maxNumber = 0;
+        int indexOfNumb = 0;
+        for (double i = 0; array.Length > i; i++)
```
тут в качестве индексатора не обязательно использовать double можно было юзать int

InTeam7 commented on Homework/Homework/Program.cs:
```diff
@@ -180,4 +183,20 @@ public static int GetElementCount(int[] arr, int itemToCount)
+        int indexOfNumb = 0;
+        for (double i = 0; array.Length > i; i++)
+        {
+            if (maxNumber < array[(int)i])
+            {
+                maxNumber = array[(int)i];
+                indexOfNumb = (int)i;
+            }
```
в принципе решение верное, только тут тебя не туда понесло, смотри как бы сделал я
```suggestion
            if (maxNumber < array[i])
            {
                maxNumber = array[i];
                indexOfNumb = i;
            }
```

danqa03 committed:
Update Homework/Homework/Program.cs

Понимаю твоё негодование, но, дал бы мне компилятор упростить (имею ввиду без обязательного указания что i - это целое число, так бы и сделал. Понятно что это мусор и так писать не нужно. Хз какой то конфликт с VS умолял прямую инициализацию ему указать

Co-authored-by: Нияз <100140390+InTeam7@users.noreply.github.com>
